Modules/QtWidgets/src/QmitkRenderWindowWidget.cpp

```diff
@@ -69,9 +69,17 @@ void QmitkRenderWindowWidget::ForceImmediateUpdate()
   mitk::RenderingManager::GetInstance()->ForceImmediateUpdate(m_RenderWindow->renderWindow());
 }
 
-void QmitkRenderWindowWidget::AddUtilityWidget(QWidget* utilityWidget)
+void QmitkRenderWindowWidget::AddUtilityWidget(QWidget* utilityWidget, UtilityWidgetPosition position)
 {
-  m_Layout->insertWidget(0, utilityWidget);
+    if (position == Top) {
+        m_VerticalLayout->insertWidget(0, utilityWidget);
+    } else if (position == Bottom) {
+        m_VerticalLayout->insertWidget(m_VerticalLayout->count(), utilityWidget);
+    } else if (position == Left) {
+        m_HorizontalLayout->insertWidget(0, utilityWidget);
+    } else if (position == Right) {
+        m_HorizontalLayout->insertWidget(m_HorizontalLayout->count(), utilityWidget);
+    }
```

Comment:
Please follow MITK style (2 spaces indent). Inserting a widget at the end of a QBoxLayout can be done with a negative index.

```suggestion
  if (position == Top) {
    m_VerticalLayout->insertWidget(0, utilityWidget);
  } else if (position == Bottom) {
    m_VerticalLayout->insertWidget(-1, utilityWidget);
  } else if (position == Left) {
    m_HorizontalLayout->insertWidget(0, utilityWidget);
  } else if (position == Right) {
    m_HorizontalLayout->insertWidget(-1, utilityWidget);
  }
```